properties-panel-extension/package.json

```diff
@@ -27,15 +27,20 @@
   "license": "MIT",
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
     "babelify": "^8.0.0",
     "grunt": "~1.0.1",
     "grunt-browserify": "^5.0.0",
     "grunt-contrib-connect": "~1.0.2",
     "grunt-contrib-copy": "~1.0.0",
     "grunt-contrib-less": "^1.0.1",
     "grunt-contrib-watch": "~1.0.0",
+    "i": "^0.3.6",
```

Comment:
What is this?

Reply:
es2015:
<img width="968" alt="Screenshot 2020-05-28 at 13 46 40" src="https://user-images.githubusercontent.com/15003836/83137577-af792e00-a0e9-11ea-8d72-9b1b13575c3a.png">

stage-0:
<img width="814" alt="Screenshot 2020-05-28 at 13 47 36" src="https://user-images.githubusercontent.com/15003836/83137645-d0da1a00-a0e9-11ea-8b09-9316f4d6088b.png">

Reply:
Removed "i" and "npm" via: https://github.com/bpmn-io/bpmn-js-examples/pull/113/commits/949e18eb9b3e161e45e03b47d4888902edec9f8e

Those are probably added by mistake 🐵

Reply:
@philippfromme